edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh

```diff
@@ -34,7 +34,7 @@ then
   if ! getent passwd "${TARGET_UID}" >/dev/null
   then
     echo "$(date --utc +"%Y-%m-%d %H:%M:%S %:z") Creating UID ${TARGET_UID} as edgeagentuser"
-    adduser -D -S -H -s /bin/sh -u "${TARGET_UID}" edgeagentuser
```

Comment:
What does -H flag do? I can't even find it on the man page

Reply:
Specifies that no home directory is created. We do need the home directory for this to work though.

-H is specific to the busybox version of adduser in docker containers, so you won't find it on the manpage for normal adduser.